### PR TITLE
fix for .env.example && example/ folder

### PR DESCRIPTION
**Pull request by KanoczTomas**

### CODE REVIEW

File: example/rpc.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require('dotenv').load()
```
Comment (dcousens) [inline example/rpc.js:1]: not required here

File: example/service.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
-let indexd = require('indexd')
+//let indexd = require('indexd')
+let indexd = require('../index')
```
Comment (dcousens) [inline example/service.js:2]: I think for the example it makes sense to do `indexd`? No?

File: example/service.js [hunk 3]
```diff
@@ -53,7 +55,7 @@ module.exports = function initialize (callback) {
       // don't add to the mempool until after a reset is complete
       if (syncQueue.running > 0) return
       if (topic !== 'hashtx') return
-      console.logTx(topic, message)
+      console.log(topic, message)
```
Comment (dcousens) [inline example/service.js:58]: :+1:

File: example/rpc.js (hunk 1 above)
Comment (KanoczTomas) [reply example/rpc.js:1]: without it I was not able to make rpc calls correctly, 8332 was used intead of the 18332 in my .env for RPC port. Where is it getting the default 8332 from than?

File: example/service.js (hunk 2 above)
Comment (KanoczTomas) [reply example/service.js:2]: oh you are right ...

File: example/rpc.js (hunk 1 above)
Comment (dcousens) [reply example/rpc.js:1]: The default is probably from https://github.com/dcousens/yajrpc/blob/master/index.js#L20

But,  `process.ENV` variables should be filled in by `require('dotenv').load()` in the `example/index.js`,  you shouldn't need to do it twice.

File: example/service.js (hunk 3 above)
Comment (dcousens) [reply example/service.js]: https://github.com/bitcoinjs/indexd/commit/85f17752166e2677d35e9c9f3b777b5104fdc38b

**Comment by dcousens** [comment]
used a45f230 instead

**Comment by dcousens** [comment]
Didn't merge and instead committed myself as I had a few other changes to rebase this on... apologies.
The credit is all yours,  so I put your name in the commit messages  507621b 8fdfd91

(I missed it in bf49b6ace14857dd5f340a7b4b55f0d417f504f9, but hopefully you are OK with that)

**Pull request closed without merging**